.github/workflows/refresh-magic-link.yml

```
@@ -0,0 +1,48 @@
+
+      - name: Load secrets
+        uses: 1password/load-secrets-action@v3
+        with:
+          export-env: true
+        env:
+          OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
+          TWITCH_CLIENT_ID: op://ci-cd/foam-staging/TWITCH_PROD_CLIENT_ID
+          TWITCH_CLIENT_SECRET: op://ci-cd/foam-staging/TWITCH_PROD_CLIENT_SECRET
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v3
```

actionlint (1.7.12)

[error] the runner of "aws-actions/configure-aws-credentials@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)